### ISSUE TITLE
Launching Eve-O preview breaks the resolution of one of the game clients

### ISSUE THREAD

**Issue by JTMMartins** [Phrynohyas/eve-o-preview#196]
This started to happen yesterday and coincided with an update of Nvidia graphic drivers and also with an update of the game client.

If I open 2 Eve clients (Eve-O is closed), everything is ok. Both in external monitor runing a 2560 X 1080. No problem.

Now I open Eve-o preview, and immediatly one of the clients gets completely weird, as if it was at half resolution, and mouse cursor position does not even matches the mouse cursor we see on screen. (we have to click in different places to activate someting).

Closing the bad client, closing eve-o preview and relaunching the client, and everything is back to normal.

In other words, I cant use eve-o preview.

Any hints ?

Thanks

Grapics card - 
Nvidia GTX 1050 TI - driver version 466.47

Eve client : latest

**Comment by JTMMartins** [comment]
Solved.

I edited the Json file, to the proper resolution, and everything seems to be back to normal.

happy.
